app/services/nomis/models/user_details.rb

```diff
@@ -13,6 +13,7 @@ class UserDetails
       attribute :status, :string
       attribute :thumbnail_id, :string
       attribute :username, :string
+      attribute :emails, :string
```

Comment:
If you don't specify a type, I think it may even make it a list for you automatically